### ISSUE TITLE
TypeError: Cannot read property 'pieceIds' of undefined

### ISSUE THREAD

**Issue by kafkiano** [apostrophecms/apostrophe-favicons#7]
Hey 

Since I've updated to v1.0.15 I get an error locally as well as on hosted environment with external Storage.

```
/home/felix/github/fullpage-cms/node_modules/apostrophe-favicons/lib/modules/apostrophe-favicons-global/index.js:167
            '_id': piece.aposFavicon.items[0].pieceIds[0]
                                              ^

TypeError: Cannot read property 'pieceIds' of undefined
    at getImage (/home/felix/github/fullpage-cms/node_modules/apostrophe-favicons/lib/modules/apostrophe-favicons-global/index.js:167:47)
    at nextTask (/home/felix/github/fullpage-cms/node_modules/apostrophe-favicons/node_modules/async/dist/async.js:5324:14)
    at next (/home/felix/github/fullpage-cms/node_modules/apostrophe-favicons/node_modules/async/dist/async.js:5331:9)
    at /home/felix/github/fullpage-cms/node_modules/apostrophe-favicons/node_modules/async/dist/async.js:969:16
    at /home/felix/github/fullpage-cms/node_modules/apostrophe-favicons/lib/modules/apostrophe-favicons-global/index.js:153:22
    at FSReqWrap.oncomplete (fs.js:135:15)

```

**Comment by kafkiano** [comment]
I think this happens only if you have no favicon defined because I've downgraded now to .14 added an image as favicon and upgraded again to .15 and apostrophe didn't crashed with these error above.

I hope this helps out

**Comment by boutell** [comment]
cc @stuartromanek, it looks like some code is making assumptions about what exists

**Comment by stuartromanek** [comment]
Yeah, being too greedy about having the module turned on but not actually using it. This is patched in `1.0.16` @felixlberg , thanks for pointing it out!

**Comment by kafkiano** [comment]
Hey Thx for fixing it... I can cofirm it.

strangely this happened sporadically but i was able to reproduce it with apostrophe palette when i changed some values there it happened instantly.

anyway thx really much for fixing it
